TESTS/mbedtls/ecp/main.cpp

```diff
@@ -20,8 +20,6 @@
 #include "utest.h"
 #include "rtos.h"
 
-using namespace utest::v1;
-
```

Comment:
I understand your point, but in our specific case, status_t was platform specific, so there won't be a use for an other `status_t`. My main concern is that if new tests will be introduced, one should also add the namsepcae prefix.
My comment wasn't a request for change, it was more of an obeservation